assets/js/modules/analytics-4/components/common/UseSnippetSwitch.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * Analytics Use Snippet Switch component.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { useCallback } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import { MODULES_ANALYTICS_4 } from '../../datastore/constants';
+import { trackEvent } from '../../../../util';
+import Switch from '../../../../components/Switch';
+const { useSelect, useDispatch } = Data;
+
+export default function UseSnippetSwitch() {
+	const useSnippet = useSelect( ( select ) =>
+		select( MODULES_ANALYTICS_4 ).getUseSnippet()
+	);
+
+	const { setUseSnippet } = useDispatch( MODULES_ANALYTICS_4 );
+	const onChange = useCallback( () => {
+		setUseSnippet( ! useSnippet );
+		trackEvent(
+			'analytics_setup',
+			useSnippet ? 'analytics4_tag_enabled' : 'analytics4_tag_disabled'
+		);
+	}, [ useSnippet, setUseSnippet ] );
+
+	if ( useSnippet === undefined ) {
+		return null;
+	}
+
+	return (
+		<div className="googlesitekit-analytics-usesnippet">
+			<Switch
+				label={ __( 'Place GA4 code on your site', 'google-site-kit' ) }
```

Comment:
Let's use "Place Google Analytics 4 code" here, to match the text in the UA snippet switch.

Reply:
Updated.